components/emoji_picker/components/emoji_picker_skin.tsx

```diff
@@ -63,7 +63,7 @@ export class EmojiPickerSkin extends React.PureComponent<Props, State> {
     extended() {
         const choices = skinsList.map((skinPair) => {
             const skin = skinPair[1];
-            const emoji = skinToneEmojis.get(skin);
+            const emoji = skinToneEmojis.get(skin)!;
```

Comment:
are there any performance implications on using one over the other? Because otherwise I'm 0/5, typescript with the right types should prevent any potential problem on both.

Reply:
From what I understand, Maps have slightly worse performance, but that might be outdated and it likely doesn't matter on the scale we'd use them. I haven't looked at that in a while though.